docs/StepbyStepManualDeploymentGuideScalarDLOnAzure.md

```diff
@@ -0,0 +1,237 @@
+# Example steps to Deploy Scalar DL on Azure
+
+This guide assumes that you create the environment on the basis of [manual deployment guide for Azure](./ManualDeploymentGuideScalarDLOnAzure.md).
+This document contains example steps for creating a Scalar DL environment in the Azure cloud.
+
+## Prerequisites
+
+You can skip the following steps If the user has an [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) or [Contributor](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#contributor) role in Azure.
+You can use the following steps to create and assign a custom role to the user. User should have `Privileged Role Administrator` or `Global Administrator` to do the following
```

Comment:
I don't understand the definition of `You` and `user`.

Who creates the resources?

>User should have `Privileged Role Administrator` or `Global Administrator` to do the following

Why does the user need a strong permissions as `Global Administrator`?

Reply:
`You` is the person who uses this document and `user` is the user account used to create the resources.

> Why does the user need a strong permissions as Global Administrator?

When the user only have custom role with the minimum permission attached to it, the user needs `Directory reader` permission to access the service principal. 
The user who assigns the permission needs `Global Administrator` permission to assign `Directory reader`.

Reply:
The Tei-san's point is that the above things are not clear from the sentences. It's important. Please fix.

Reply:
> You is the person who uses this document and user is the user account used to create the resources.

I think the person who creates resources will read the doc. So, if your definition is correct,  you == user mostly, which I don't think is correct.
Be specific about the actors and who is doing what.

Reply:
Updated sentences in 8f9b0ff